pyproject.toml

```diff
@@ -1,29 +1,40 @@
-[tool.poetry]
+[project]
 name = "aioswitcher"
 version = "6.0.1-dev"
-description = "Switcher Python Integration."
-license = "Apache-2.0"
-authors = [ "Tomer Figenblat <tomer@tomfi.info>" ]
-maintainers = [ "Shay Levy" ]
+description = "PyPi module integrating with various Switcher devices"
+license = { text = "Apache-2.0" }
 readme = "README.md"
+requires-python = ">=3.12.0,<4.0"
+authors = [
+    { name = "Tomer Figenblat", email = "tomer@tomfi.info" }
+]
+maintainers = [
+    { name = "Shay Levy" },
+    { name = "Yogev Bokobza" }
+]
+keywords = [ "home-automation", "smarthome", "switcher" ]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Topic :: Home Automation",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Typing :: Typed"
+]
+dependencies = [
+    "pycryptodome==3.18.0",
+    "aiohttp==3.10.3",
```

Comment:
Is it possible to pin the requirements to a minimum only? HA is already at `aiohttp==3.11.11` (and constantly changing as it is maintained by HA member)
This is what we had before the change:
```
  pycryptodome = ">=3.18.0"
  aiohttp = ">=3.10.3"
```

Reply:
I usually prefer pinning dependency versions, this makes for a more robust solution. And it should be ok if we keep dependabot working and release frequently. But I understand the volatile situation in this case.

I'll do what you asked, but I'll also block major bumps:

```
pycryptodome = ">=3.18.0,<4.0.0"
aiohttp = ">=3.10.3,<4.0.0"
```

Reply:
https://github.com/TomerFi/aioswitcher/pull/829/commits/433f1b40f9160df2c3d17bfffdb5d3a6fb4be65d

Reply:
Yes, this is perfect, when a major bump will happen means we need to review breaking changes and decide. This will be a long process for others anyhow so we won't block anything. Thanks!